README.md

```diff
@@ -213,13 +212,6 @@ bring up discussion topics, and ask questions.
 You can subscribe to PlasmaPy's low-volume [mailing list] to receive
 PlasmaPy newsletters and other announcements.
 
-### Suggestion box
-
-We have a [suggestion box] if you would like to (optionally
-anonymously) suggest a feature/topic for consideration. These
-suggestions might be changed into GitHub issues for further
-discussion.
-
```

Comment:
Removing this since it's been pretty rare for us to get suggestions in the suggestion box.